gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
Thought I'd seen it before. Guess it doesn't hurt, but its inclusion might kinda imply we tested it.